src/ameba/ast/visitors/implicit_return_visitor.cr

```diff
@@ -280,6 +285,14 @@ module Ameba::AST
       false
     end
 
+    def visit(node : Crystal::Block) : Bool
+      report_implicit_return(node)
+
+      node.body.accept(self)
```

Comment:
what about other parts? args/call?

Reply:
Unless I missed something, only the body can contain nodes that could be unused.